cfg/validate_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2024 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cfg
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func validLogRotateConfig() LogRotateLoggingConfig {
+	return LogRotateLoggingConfig{
+		BackupFileCount: 0,
+		Compress:        false,
+		MaxFileSizeMb:   1,
+	}
+}
+
+func TestValidateConfig(t *testing.T) {
+	testCases := []struct {
+		name                   string
+		config                 *Config
+		expectedCustomEndpoint string
```

Comment:
I think it's good enough to check whether Validate returns an error. This expectedCustomEndpoint doesn't add any value and can be removed.

Reply:
Added a test case where this is being used. PTAL.